composer.json

```diff
@@ -23,11 +23,6 @@
     "config": {
         "preferred-install": "dist"
     },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "3.4-dev"
```

Comment:
Why is this removed?

Reply:
Fixes. Was a local Charger i forgot about